tests/e2e/ansible/install_test_deps.yml

```diff
@@ -20,6 +20,8 @@
                 repo: https://github.com/bats-core/bats-core.git
                 dest: bats-core
                 force: yes
+                retries: 3
```

Comment:
`Install test dependencies` can be retried too I think.

Reply:
good catch